packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -218,6 +221,8 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
   }
 
   protected buildOperation(node: OperationDefinitionNode, documentVariableName: string, operationType: string, operationResultType: string, operationVariablesTypes: string): string {
+    operationResultType = `${this._prefix}${operationResultType}`;
```

Comment:
`buildOperation` should get the built `operationResultType`, and not build it.

Reply:
@dotansimha should we add the `prefix` here so it will be converted prior to sending as arguments.
https://github.com/dotansimha/graphql-code-generator/blob/master/packages/plugins/other/visitor-plugin-common/src/client-side-base-visitor.ts#L340